js/app/modules/contentGroup/mediaLightbox.js

```diff
@@ -57,6 +57,8 @@ const MediaLightbox = new Module.Class({
     },
 
     _on_close: function () {
+        this.lightbox_widget.destroy();
```

Comment:
You should still protect yourself from destruction coming from other layers — e.g. one of the ancestors being destroyed.